lib/ProductOpener/Display.pm

```diff
@@ -435,6 +435,33 @@ sub process_template($template_filename, $template_data_ref, $result_content_ref
 }
 
 
+=head2 redirect($status_code, $redirect_url)
```

Comment:
to @yuktea  : typically a function that, when we will split is part of "http utils" more than display.